sources/Outputs/src/Dyd.cpp

```diff
@@ -231,5 +267,15 @@ Dyd::writeGenConnect(const algo::GeneratorDefinition& def, unsigned int index) {
   return {connection, signal};
 }
 
+void
+Dyd::writeHvdcLineConnect(const boost::shared_ptr<dynamicdata::DynamicModelsCollection>& collection, const algo::HvdcLineDefinition& hvdcLine) {
+  if (hvdcLine.position == algo::HvdcLineDefinition::Position::FIRST_IN_MAIN_COMPONENT) {
+    collection->addConnect("NETWORK", hvdcLine.converter1.busId + "_ACPIN", hvdcLine.id, "hvdc_terminal1");
+    collection->addConnect("NETWORK", hvdcLine.converter2.busId + "_ACPIN", hvdcLine.id, "hvdc_terminal2");
+  } else if (hvdcLine.position == algo::HvdcLineDefinition::Position::SECOND_IN_MAIN_COMPONENT) {
+    collection->addConnect("NETWORK", hvdcLine.converter1.busId + "_ACPIN", hvdcLine.id, "hvdc_terminal2");
+    collection->addConnect("NETWORK", hvdcLine.converter2.busId + "_ACPIN", hvdcLine.id, "hvdc_terminal1");
+  }
```

Comment:
Add a comment for the else case, for readability

Reply:
What do I put in this else case ?

Reply:
nothing just a comment or a TODO for future devs